cloud_chatops/src/pr_reminder.py

```diff
@@ -0,0 +1,205 @@
+"""This module handles the posting of messages to Slack using the Slack SDK WebClient class."""
+
+from typing import List, Dict
+from datetime import datetime, timedelta
+from slack_sdk import WebClient
+from slack_sdk.errors import SlackApiError
+from src.read_data import get_token, get_user_map, get_repos
+from src.get_github_prs import GetGitHubPRs
+
+
+class PostPRsToSlack:
+    """
+    This class handles the Slack posting.
+    """
+
+    def __init__(self):
+        super().__init__()
+        self.repos = get_repos()
+        self.client = WebClient(token=get_token("SLACK_BOT_TOKEN"))
+        self.slack_ids = get_user_map()
+        self.prs = GetGitHubPRs(get_repos(), "stfc").run()
+        self.channel = "temp-chatops"
+
+    def run(self, mention, channel=None) -> None:
```

Comment:
this should be the only public facing method in the class - all others should be private since they should only be used internally

Reply:
Rule of thumb is if you want to integrate this class into other automation code - what methods should make up the API and what should we not let other code outside the class access